docs/source/scripts/build_opsets.py

```diff
@@ -0,0 +1,79 @@
+import os
+from pathlib import Path
+
+import torch
+from collections import OrderedDict
+from torchgen.gen import parse_native_yaml
+import torch._prims as prims
+
+ROOT = Path(__file__).parent.parent.parent.parent
+NATIVE_FUNCTION_YAML_PATH = ROOT / Path("aten/src/ATen/native/native_functions.yaml")
+TAGS_YAML_PATH = ROOT / Path("aten/src/ATen/native/tags.yaml")
+
+print(Path(__file__))
+print(ROOT)
+print(NATIVE_FUNCTION_YAML_PATH)
+print(TAGS_YAML_PATH)
```

Comment:
logging? Probably don't want this to go to stdout?

Reply:
This is debugging log, will remove before check in.

Reply:
These prints produces 

```
source/scripts/build_opsets.py
.
aten/src/ATen/native/native_functions.yaml
aten/src/ATen/native/tags.yaml
```

I am wondering how the doc build CI is set up? It looks like only the doc folder is present in the docker's workspace. I need to access "aten/src/ATen/native/native_functions.yaml" file to check the canonical tag. Where can I find this file in the docker env?

Reply:
sorry, i don't know off the top of my head. Maybe try with-ssh/